src/platform/plugins/shared/workflows_extensions/common/steps/ai/temp.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { jsonSchemaToZod } from '@n8n/json-schema-to-zod';
+import type { JSONSchema7 } from 'json-schema';
+import { z } from '@kbn/zod/v4';
+
+/**
+ * Converts a JSON Schema string type to a Zod string schema
+ */
+function convertStringSchema(jsonSchema: JSONSchema7): z.ZodType {
```

Comment:
Has to be removed and replaced by API in https://github.com/elastic/kibana/pull/244223 is merged